src/harness/certs/generate_fake_certs.sh

```diff
@@ -139,18 +139,17 @@ openssl ca -cert sas_ca.cert -keyfile private/sas_ca.key -in sas_1.csr \
     -policy policy_anything -extensions sas_client_mode_req_sign -config ../../../cert/openssl.cnf \
     -batch -notext -create_serial -utf8 -days 1185 -md sha384
 
-# Generate normal operation client certificate/key.
-echo -e "\n\nGenerate generic 'client' certificate/key based on domainProxy"
+# Generate normal operation device certificate/key.
+echo -e "\n\nGenerate 'certs for devices' certificate/key"
 openssl req -new -newkey rsa:2048 -nodes \
-    -reqexts oper_req -config ../../../cert/openssl.cnf \
-    -out client.csr -keyout client.key \
-    -subj "/C=US/O=Wireless Innovation Forum/OU=WInnForum Domain Proxy Certificate/CN=0123456789:0001"
-openssl ca -cert proxy_ca.cert -keyfile private/proxy_ca.key -in client.csr \
-    -out client.cert -outdir ./root \
-    -policy policy_anything -extensions oper_req_sign -config ../../../cert/openssl.cnf \
+    -reqexts cbsd_req -config ../../../cert/openssl.cnf \
+    -out device.csr -keyout device.key \
+    -subj "/C=US/O=Wireless Innovation Forum/OU=WInnForum CBSD Certificate/CN=test_fcc_id:test_serial_number"
```

Comment:
I think you need to change CN=test_fcc_id:test_serial_number" to CN="test_fcc_id_a:test_serial_number_a"
or create new device with fcc_id="test_fcc_id" and serial_number="test_serial_number" and change all security testcases because all these tests use device_a

Reply:
This 'device.cert' was supposed to be the default generic certificate for cbsd, that is not only used for device_a, but potentially for any device.

I consider the CN part as purely informative: if we want to 'lock' a certificate to a given CBSD we have to use the OID FCCID and SERIAL and not be tied on a particular formating of the CN.
Because these OID are optional in the spec, it is possible to define this kind of 'generic' certificate.
Do you agree with this?

To go back to this 'device.cert', as it seems used only for SCS tests, I did the following:
    - remove the 'device.cert'
    - in sas.py, remove GetDefaultCbsdSSLCertPath() + Key 
    - in security_testcase.py: in doCbsdTestCipher(): clearly indicate that the default cert is 'device_a'  
    - update the generation of device_a and device_c by adding the OID FCCID and SERIAL